jupyterhub_chameleon/spawner.py

```diff
@@ -67,18 +65,14 @@ def _name_template(self):
     @property
     def volumes(self):
         vols = {}
-        artifact = self.get_artifact()
-        if not (artifact and artifact.ephemeral):
-            if self.name:
-                vols[self._named_name_template] = self._gen_volume_config(self.work_dir)
-                # Also mount the default directory as shared dir
-                # vols[self._default_name_template] = self._gen_volume_config(
-                #    self.share_dir
-                # )
-            else:
-                vols[self._default_name_template] = self._gen_volume_config(
-                    self.work_dir
-                )
+        if self.name:
```

Comment:
So we're getting rid of this "ephemeral" ability? It was there for a reason - it's a mechanism for launching an artifact server that does not persist the work dir to our central storage. This can be handy as it means your changes get wiped away next time you launch, giving you a clean slate.